src/components/fields/MultipleChoice.js

```diff
@@ -98,13 +99,13 @@ class MultipleChoice extends AskField {
         style={this.getOptionStyle(i)}
         key={i}
         ><input
-          style={styles.optionCheck}
-          onClick={this.onClick.bind(this, i)}
-          tabindex='0'
-          name={!this.props.multipleChoice ? this.props.title : false}
-          type={this.props.multipleChoice ? 'checkbox' : 'radio'}
-          key={i} />{option.title}</label>
-    })
+            style={ styles.optionCheck }
```

Comment:
:P 
There are some linted & unlinted things mixed from merging the linted master to this branch, it had looooots of conflicts, that's why I want to finish the linting after merging this.